rest-server/src/routes/index.js

```diff
@@ -19,17 +19,18 @@
 // module dependencies
 const express = require('express');
 const controller = require('../controllers/index');
-const authRoute = require('./auth');
-const jobRoute = require('./job');
-
+const tokenRouter = require('./token');
+const userRouter = require('./user');
+const jobsRouter = require('./jobs');
```

Comment:
do not need to rename filename

Reply:
Any principle to name the module? `jobs.js` here is just a router, not a class, why do we need to use `job.js`?

Reply:
the principle is to use singular form and be consistent.

Reply:
do you mean all .js files should be organized by module names? modules include: token, user, job. my understanding correct? Then how do you feel about the inconsistency of the api endpoint name (plural form) and the module name (singular form as you suggested)?